script/set_bs_employer_profile_id_on_staff_role.rb

```diff
@@ -0,0 +1,45 @@
+def profile_fein_for_role(role)
+  old_profile_id = role.employer_profile_id.to_s
+  ::EmployerProfile.find(old_profile_id).try(:fein)
+end
+
+feins = [ "050518534", "030588648", "041617630", "041810570", "042967119", "043399205",
```

Comment:
Can we get rid of this hard coded feins?

Reply:
@sri49 Added feins to script because this is one-time used script to fix employers staff roles who signed up with old url.